### DISCUSSION TITLE
Zen Cart 2.1.x

### DISCUSSION THREAD

**Discussion by scottcwilson** [zencart/zencart#6629] [General]
We're getting ready to start test builds of 2.1.0.  You can read the What's New content here: 

https://docs.zen-cart.com/release/whatsnew_2.1.0/

**Comment by scottcwilson** [comment]
2.1.0-alpha1 build may be downloaded from https://github.com/zencart/zencart/releases/tag/v2.1.0-alpha1

**Comment by scottcwilson** [comment]
The next alpha build will be done the week of August 25th.  We have some nice new changes since alpha1 like the conversion of zc_install to use Bootstrap, and a number of good ones on deck that will hopefully make it into this build. 

If you have made a change that I missed in the release notes, please let me know and I'll get it in.  Thanks to all contributors.

https://docs.zen-cart.com/release/whatsnew_2.1.0/

**Comment by torvista** [comment]
I think more shouting is in order:

> < blink equivalent>NEW FEATURE: ZEN CART NOW INCLUDES STOCK CONTROL PER ATTRIBUTE/VARIANT< /blink equivalent>...and then the sober text

**Comment by scottcwilson** [comment]
POSM has pride of place as the first New Feature, but I am certainly happy to bold this text to draw more attention to it. 

Thanks to Cindy for generously contributing her work.

**Reply by torvista** [reply]
My opinion is that as a token of appreciation, it would not be amiss to point out that the Price/Weight add-on to POSM
https://vinosdefrutastropicales.com/index.php?main_page=product_info&cPath=2_7&products_id=60
exists, and will, as a bonus, replace the much-hated (by anyone trying to maintain it)/unsupported Hydra that is Dynamic Price Updater.

**Reply by scottcwilson** [reply]
There is already a link to POSM-PW on the POSM page. 

https://docs.zen-cart.com/user/running/posm/#b-using-the-posm-pw-plugin

**Reply by torvista** [reply]
Good job!

**Comment by scottcwilson** [comment]
Next 2.1.0 alpha build scheduled for this weekend.  Thanks to all contributors. 

Notable improvements since last build (2.1.0-alpha1, Aug 4, 2024)
- Search in Options Values Manager fixed
- For robustness, missing language constants will now fall back to their English definitions
- Featured Categories
- Layout Boxes Controller page UI improved
- AWESOME new installer

The complete list is here: 
https://docs.zen-cart.com/release/whatsnew_2.1.0/

**Comment by scottcwilson** [comment]
Build 2.1.0 alpha2 is now ready. 

https://github.com/zencart/zencart/releases/tag/v2.1.0-alpha2

**Comment by scottcwilson** [comment]
Beta1 build coming in the next few days.  Stay tuned!

**Comment by scottcwilson** [comment]
Release build scheduled for this weekend.
